docs/src/tutorials/simple_poisson_process.md

```diff
@@ -6,6 +6,12 @@ primarily in chemical or population process models, where several types of jumps
 may occur, can skip directly to the [second tutorial](@ref ssa_tutorial) for a
 tutorial covering similar material but focused on the SIR model.
 
+!!! note
+    
+    If you have a background in point process theory, we recommend that
+    you read our example on [modelling temporal point processes with
+    JumpProcesses](@ref tpp_tutorial)
+
```

Comment:
I would just merge this into the first paragraph for parallel construction and say something as the new last sentence like "Readers with a background in point process theory can skip directly to [your new tutorial]."

Reply:
Fixed.